data/config_files/example_config_controlled.json

```diff
@@ -101,6 +101,19 @@
       "focalLength": 50.0
     }
   },
+  "occlusion":{
+    "render-occlusion": false,
+    "__comment-occlusionMode-1": "The options for the occlusionMode are:",
+    "__comment-occlusionMode-2": "eyes - This mode retrieves the location of the eyes from the provided .tlms file and renders black circles over both eyes on the face.",
```

Comment:
where does this tlms file come from? how can I provide it? or are internally generated landmarks used?